rim-page.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rim</title>
+        <style>
+            .container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: float;
+            }
+            .column {
+                width: 25%;
+                float: left;
+            }  
+            ul {
+                list-style-type: none;
+            }
+            li {
+                display: inline;
+            }
+            .col-2 {
+                width: 10%;
+                float: left;
+            }
+            .hero-image {
+                height: 100%;
+                background-image: url("fistbump-over-desk.jpg");
+                background-position: center;
+                background-repeat: no-repeat;
+                background-size: cover;
+                position: relative;
+            }
+            .hero-text {
+                text-align: left;
+                position: absolute;
+                top: 50%;
+                bottom: 50%;                
+            }                       
+            .col-3 {
+                width: 33.3%;
+                float: left;
+            }            
+            .col-4 {
+                width: 50%;
+                float: left;
+            }
+            .col-5 {
+                width: 20%;
+                float: left;
+            }
+            .copyright {
+                display: block;
+                width: 100%;
+                float: right;
+            }    
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="col-2">
```

Comment:
This should just be column.
You have given col-2 a width of 25% in your css.
However, this is only useful if you intend for this row to contain 4 columns.
If you only have one column in a row, you would just give it a column class that has 100% width (i.e. - column takes up full width of page).
In this case, we are using "column" as a stand-in class for a lot of things.
But here, just use column and know that this would have to be a class with column, that has a 100% width.